go2_ws/src/go2_description/xacro/leg.xacro

```diff
@@ -19,7 +16,7 @@
             <xacro:if value="${(mirror_dae == True) and (front_hind_dae == False)}">
                 <origin rpy="0 0 0" xyz="${-leg_offset_x} ${leg_offset_y} 0"/>
             </xacro:if>
-            <parent link="trunk"/>
+            <parent link="base_link"/>
```

Comment:
Is changing the parent of `${name}_hip_joint` from `trunk` to `base_link` necessary?

I think it's okay to keep the parent as `trunk` (this including the parent of certain links in `robot.xacro`), as I believe the `trunk` in the xacro fie corresponds to the `base` in the URDF file. Their dae mesh files both correspond to the Go2 main body.

Reply:
Yes, please remove the `trunk` links in all Xacro files if possible. Removing it can help prevent future confusion.